specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2024-01-01-preview/workspaceRP.json

```diff
@@ -1888,6 +3034,28 @@
       },
       "x-ms-discriminator-value": "CustomKeys"
     },
+    "DefaultResourceProvisioningState": {
+      "enum": [
+        "NotStarted",
+        "Failed",
+        "Creating",
+        "Updating",
+        "Succeeded",
+        "Deleting"
+      ],
+      "type": "string",
+      "readOnly": true,
+      "x-ms-mutability": [
+        "read"
+      ],
+      "x-ms-enum": {
+        "name": "DefaultResourceProvisioningState",
+        "modelAsString": true
+      }
+    },
+    "DeploymentModelVersionUpgradeOption": {
```

Comment:
>DeploymentModelVersionUpgradeOption [](http://example.com/codeflow?start=5&length=35)

Empty/unschematized object pattern is not allowed. Please provide the schema for this object.